preview/Win7Msix/Win7MSIXInstaller/MsixRequest.hpp

```diff
@@ -74,6 +81,17 @@ class MsixRequest
         return true;
     }
 
+    bool GetIsCancelClicked()
+    {
+        return m_isInstallCancelled;
+    }
+
+    //Setters
+    void SetIsCancelClicked()

```

Comment:
>SetIsCancelClicked [](start = 9, length = 18)

could just be Cancel() or CancelRequest() as a verb.

Reply:
Modified method name to CancelRequest().